### ISSUE TITLE
Update sever address

### ISSUE THREAD

**Issue by sgrieve** [oracc/nammu#267]
Following Steve's email:

```
Morning, all,

I have recompiled and reinstalled the server software to use build-oracc.museum.upenn.edu as its name.

Could you change the default server which Nammu communicates with to the same?

I will give people a week or so to update their apps: let's say that I will ask our IT guys to reset oracc.museum.upenn.edu to be a synonym for oracc.iaas.upenn.edu (the new cloud server) on Monday August 21st?  I am in town all of that week and the start of the next, but I will be off the servers from about the 30th Aug to 4th Sep inclusive.

Thanks very much and all the best,

 Steve
```

We need to update the server address in the default settings for Nammu.

**Comment by sgrieve** [comment]
We are thinking about moving the server settings to a new file, as they probably don't need to be edited by the typical user.

**Comment by sgrieve** [comment]
This will probably be a breaking change so we will hold off on this update until the release of version 1.0 to make life easier.

**Comment by sgrieve** [comment]
The new server is now working and the current version has both the old and new versions of the server available in the settings pane.

A patching method will be written on Monday to resolve this prior to the official release of version 1.0.0

**Comment by sgrieve** [comment]
Resolved in #292
